<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenSprinkler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="btnDownload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="toolTip">
     <string>Download Firmware</string>
    </property>
    <property name="text">
     <string>Download Firmware</string>
    </property>
    <property name="shortcut">
     <string>Meta+F</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Detect Hardware</string>
    </property>
    <property name="text">
     <string>Detect Hardware</string>
    </property>
    <property name="shortcut">
     <string>Meta+D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>260</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDevice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>102</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hardware:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbDevice">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblFirmware">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Firmware:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbFirmware">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Upload Selected Firmware</string>
    </property>
    <property name="text">
     <string>Upload Selected Firmware</string>
    </property>
    <property name="shortcut">
     <string>Meta+U</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionUpdate_Firmware">
   <property name="text">
    <string>Update Firmware</string>
   </property>
  </action>
  <action name="actionDetect_Device">
   <property name="text">
    <string>Detect Device</string>
   </property>
  </action>
  <action name="actionHow_To">
   <property name="text">
    <string>How To</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
